dev/io.openliberty.wsoc.internal_fat/fat/src/io/openliberty/wsoc/endpoints/client/basic/IdleTimeoutTCKClientEP.java

```diff
@@ -100,8 +103,14 @@ public void onClose(Session session, CloseReason closeReason) {
 
     @OnError
     public void onError(Session session, java.lang.Throwable throwable) {
-
-        _wtr.addExceptionAndTerminate("Error during wsoc session", throwable);
+        // Client automatically throws an error when a 1006 response is added. Verify if this
+        // is what is happening on this client and do not throw error when it does since as seen above
+        // on the onClose we expect it to behave this way and need to verify it
+        LOG.warning(throwable.toString());
```

Comment:
Since I have a better understanding now, this line might not be necessary. Up to you if you want to leave it there or not.